### PR TITLE
SO-4731: Update ECL to version 1.5

### PR DESCRIPTION
**Pull request by apeteri**
Functionality from QL is included in the update.

### CODE REVIEW

File: snomed/com.b2international.snowowl.snomed.datastore/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -21,16 +21,14 @@ Require-Bundle: org.eclipse.core.runtime;bundle-version="3.9.0",
  com.b2international.mapdb;bundle-version="[3.0.7,4.0.0)",
  com.b2international.snowowl.snomed.icons;bundle-version="6.15.0",
  com.b2international.snowowl.core;visibility:=reexport,
- com.b2international.snomed.ecl;visibility:=reexport,
- com.b2international.snomed.ql;visibility:=reexport
+ com.b2international.snomed.ecl;visibility:=reexport
```
Comment (cmark) [inline snomed/com.b2international.snowowl.snomed.datastore/META-INF/MANIFEST.MF:24]: I'm not seeing any version bumps for the ECL 1.5 language (in the root `pom.xml` for example). Am I missing something?

**Review by cmark — CHANGES_REQUESTED** [review]
Looks great so far! Resolve the issues in the https://github.com/b2ihealthcare/snomed-ecl/pull/4 PR first, then here. We will merge the language PR first then this.

**Comment by cmark** [comment]
@apeteri could you please resolve the conflict? Thanks!

**Comment by apeteri** [comment]
This is the commit that introduces `com.b2international.snowowl.snomed.core.request` to the exported package list: https://github.com/b2ihealthcare/snow-owl/commit/fcf89c5074ebdfabcb9b47f028c85d102544f9c5

I haven't found anything that uses the concrete classes outside of `SnomedPlugin` (which registers them as a query optimizer / concept search request evaluator implementation) at the moment, so I have removed it along with the QL package.

**Review by cmark — APPROVED** [review]
LGTM!

**Pull request merged**